entities/jackson/src/main/java/org/ccci/gto/globalreg/entity/jackson/EmailAddress.java

```diff
@@ -16,22 +17,71 @@
     "location",
     "primary",
     "parent_id",
-    "client_integration_id"
+    "client_integration_id",
+    "created_by",
+    "updated_at",
+    "user_entered",
+    "client_updated_at"
 })
 public class EmailAddress {
 
     @JsonProperty("id")
     private String id;
     @JsonProperty("email")
-    private String email;
+    private Email email;
```

Comment:
changes like this are part of the reason I feel these objects living in the API client itself may be inappropriate. If this had been in production already and we all of a sudden change object signatures in a backwards incompatible way it will cause breakage for applications depending on this API.